back-end/db/utils.ts

```diff
@@ -28,10 +28,10 @@ const chooseRandomFromList = (listOfSomething: any[]): any => {
 
 const selectRandom = (mainList, newList, count) => {
     let mainCopy = [...mainList];
-
+    let newList_lower = newList.map(newList => newList.toLowerCase());
```

Comment:
very small change for this line - when you use `newList.map`, the argument passed is a function that should take in one element, and return the updated element. Your code works as intended, but the variable name of `newList` within the arrow function is a little confusing - it could be `item` or `listElement` or something like that. Basically, you're calling the `toLowerCase` function on a _string_, not a list... so doing `newList.toLowerCase` just looks a little weird.

Hope that makes sense!